scripts/tests/backend-leave-fed-test.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```

Comment:
I don't think this script is necessary, the gateway leaving test should be the same regardless of whether the lightning node is mocked or not